pkg/node/compute.go

```diff
@@ -197,9 +199,9 @@ func NewComputeNode(
 	if managementProxy != nil {
 		// TODO: Make the registration lock folder a config option so that we have it
 		// available and don't have to depend on getting the repo folder.
-		repo, _ := pkgconfig.Get[string]("repo")
+		computeDataPath, _ := pkgconfig.Get[string](cfg, types.NodeComputeDataPath)
 		regFilename := fmt.Sprintf("%s.registration.lock", nodeID)
-		regFilename = filepath.Join(repo, pkgconfig.ComputeStorePath, regFilename)
+		regFilename = filepath.Join(computeDataPath, regFilename)
```

Comment:
we want to minimize number of paths the users have to define in the config, and follow more convtions over configurations. The idea is to allow users to only define the repo path, and let each component create its sub-directories and files under the repo instead of allowing users to configure those as well

Reply:
I am in agreement so long as we make it the responsibility of the repo to create and manage these paths on disk, or whatever medium is backing the repo.